rs/bitcoin/adapter/src/lib.rs

```diff
@@ -49,14 +51,12 @@ mod transaction_store;
 // malicious fork can be prioritized by a DFS, thus potentially ignoring honest forks).
 mod get_successors_handler;
 
-pub use blockchainmanager::BlockchainManager;
+use crate::{
+    common::BlockHeight, config::IncomingSource, router::start_main_event_loop, stream::StreamEvent,
+};
 pub use blockchainstate::BlockchainState;
```

Comment:
Don't you also want to hide this?

Reply:
It is used in the benches, will do it in a followup PR